RD_WorkShop1/index.html

```diff
@@ -0,0 +1,194 @@
+                                    <div class="star"></div>
+                            </div> 
+                        </div>
+                    </div>
+
+                    <div class="container-testymonials">
+                        <div class="inner-block">
+                            <div class="card">
+                                <h3>Courtney Henry</h3>
+                                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+                                <div class="star"></div>
+                            </div> 
+                        </div>
+                    </div>
+
+                    <div class="container-testymonials">
+                        <div class="inner-block">
+                            <div class="card">
+                                <h3>Courtney Henry</h3>
+                                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+                                <div class="star"></div>
+                            </div> 
+                        </div>
+                    </div>
+
+                    <div class="container-testymonials">
+                        <div class="inner-block">
+                            <div class="card">
+                                <h3>Courtney Henry</h3>
+                                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+                                <div class="star"></div>
+                            </div> 
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="section-4">
+            <div class="contact-us">
+                <h1>Contact Us</h1>
+            </div>
+
+            <div class="form-icon">
+                <div class="input">
+                    <input type="text" class="form-control input-cont" placeholder="name">
+                    <input type="text" class="form-control input-cont" placeholder="(303) ">
+                    <input type="email" class="form-control input-cont" placeholder="e-mail">
+
+                    <div class="send-button">
+                        <button type="button" class="btn btn-primary btn-lg">Send</button>
+                    </div>
+                </div>
+    
+                <div class="icon">
```

Comment:
on small screens this block overlap the footer

<img width="506" alt="image" src="https://github.com/GeoDevH/Robot_Dreams/assets/13466381/e761c115-34d5-4429-b0b0-6d542ce624a9">